WeakAuras/Transmission.lua

```diff
@@ -1675,6 +1675,18 @@ function WeakAuras.ImportString(str)
   end
 end
 
+function crossRealmSendCommMessage(prefix, text, chattype, target, queueName, callbackFn, callbackArg)
+  if chattype == "WHISPER" and target and not UnitIsSameServer(target) then
+    if UnitInRaid(target) then
+      chattype = "RAID"
+    elseif UnitInParty(target) then
+      chattype = "PARTY"
+    end
```

Comment:
Shouldn't that do:
else
 return

?

Reply:
yes

Reply:
If that happen the whole PR can be reverted

Reply:
We might not immediately recognize that. If the code would do what I suggest, it would just magically start working.